### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by holm**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +48,7 @@
     "validate.io-nonnegative-integer": "^1.0.0",
     "validate.io-number-primitive": "^1.0.0",
     "validate.io-object": "^1.0.4",
-    "validate.io-positive-primitive": "^1.0.0",
+    "validate.io-positive": "^1.0.0",
```
Comment (kgryte) [inline package.json]:
```suggestion
    "validate.io-positive-primitive": "^1.0.0",
```

Comment (kgryte) [reply package.json]: This should not be changed.

**Review by kgryte — CHANGES_REQUESTED** [review]
Thanks for submitting this PR!

**Comment by kgryte** [comment]
**Aside**: you may also be interested in [stdlib](https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/stats/base/dists/beta/pdf), the successor to this project. See the link for the beta distribution PDF.

File: package.json (hunk 1 above)
Comment (holm) [reply package.json]: Why not? This library uses `validate.io-positive`, not `validate.io-positive-primitive` (https://github.com/distributions-io/beta-pdf/blob/master/lib/validate.js#L6). Even the test fails before the changes in this PR.

Comment (kgryte) [reply package.json]: Ah...then the `require` path in `validate.js` should be changed, instead.

**Comment by kgryte** [comment]
@Planeshifter Once the changes in this PR are merged, would you mind doing a patch release?

**Review by kgryte — APPROVED** [review]
@holm Thanks!

**Comment by kgryte** [comment]
@Planeshifter Should be ready for a patch release to `npm`.

**Comment by papandreou** [comment]
@Planeshifter ping? Would be really nice to get this released :)

**Comment by kgryte** [comment]
Use the `stdlib` version instead: https://github.com/stdlib-js/stats-base-dists-beta-pdf.

**Comment by kgryte** [comment]
@Planeshifter You may want to still do a patch release of this package, given its support for typed arrays and matrices.

**Comment by papandreou** [comment]
@kgryte, thanks, I'll try that instead!

**Pull request merged**